crt_portal/cts_forms/forms.py

```diff
@@ -39,6 +39,9 @@
 
 logger = logging.getLogger(__name__)
 
+def _add_empty_choice(choices):
+    """Add an empty option to list of choices"""
+    return (('', '-----'),) + choices
```

Comment:
Nitpicky but our content source of truth has the default as `- Select -`. If we change this string to follow that convention, we can use this helper in the `LocationForm` class; we are doing a similar transform of the choices tuple there too.

Reply:
Ooo! Not nitpicky at all, thanks for pointing this out!

Will remedy shortly